src/create.ts

```diff
@@ -61,13 +63,19 @@ export async function create(options: CreateOptions): Promise<Readable> {
 				},
 			},
 		},
-	})) as unknown as ReleaseManifest;
+	})) as SDK.Release;
```

Comment:
I think you might be able to drop this type cast here and instead change the following for the same effect:

```typescript
const remoteRelease = (await sdk.pine.get<SDK.Release>({
...
```

Reply:
It does look cleaner that way.